src/styles.scss

```diff
@@ -1,97 +1,15 @@
 /* You can add global styles to this file, and also import other style files */
 
-@import "~foundation-sites/scss/foundation";
+@import '~foundation-sites/scss/foundation';
 @include foundation-everything($prototype: true);
 @include foundation-flex-classes;
 @include foundation-flex-grid;
 
+@import 'ionicons';
 
-// kashti sidebar styles
+@import 'global_variables';
 
-#sidebar-wrap {
-  flex: 0 0 56px;
-  margin: 0;
+.full-height {
+  flex: 0 0 3.5em;
```

Comment:
@flynnduism I changed this from `#sidebar-wrap` to a class in an attempt to be semantic. I know this class ensures the element will use the full height of the browser. I wonder if we could extract the flex property to it's own class that's meaningful semantically. 

thus the resulting HTML would be something like

```html
<div class="column full-height flex-right-margin-10" data-off-canvas data-position="left">
    <div id="sidebar">
```
(`flex-right-margin-10` doesn't really make sense...not sure how to describe the flex property here semantically)

Reply:
👍 

Changing the id to a class is better.
Splitting the .full-height out to be either a class of it's own or a mixin makes sense. 

Is `flex-right-margin-10` to make re-usable gutter elements?

Reply:
> Is flex-right-margin-10 to make re-usable gutter elements?

Yeah...I was thinking about how to convey (in English) what this does:

```css
flex: 0 0 3.5em;
```

In this case, does the above specify the initial main size of a flex item?

Reply:
This the flexbox equivalent to width for the column. Modularize it any way you want, it's a singular exception to the existing grid so not a common element.